File: Java/test/util/TestUtil.java
package util;

import java.util.Random;

/**
 * Created by Chris Keyser on 10/3/2016.
 */
public class TestUtil {
    public static final int ARR_SIZE = 1000;

    private static Random rand = new Random();

    /**
     * generates an array of random numbers of n length
     * currently generates numbers >= 0 because of radix sort
     */
    public static int[] generateRandomArray(int len) {
        int[] arr = new int[len];

        for(int i = 0; i < len; i++) {
            arr[i] = Math.abs(rand.nextInt());
        }

        return arr;
    }
}

File: Scala/src/test/util/TestUtil.scala
package util

import scala.util.Random

/**
  * Created by Chris Keyser on 10/2/2016.
  */
object TestUtil {
    val ARR_SIZE = 1000

    def RandomArray(len: Int): Array[Int] = {
        val arr = Array[Int]()
        val rand = new Random()

        for(i <- Range(0, len)) {
            arr :+ rand.nextInt()
        }

        arr
    }
}
